ui/components/app/collectible-default-image/collectible-default-image.js

```diff
@@ -9,14 +9,17 @@ export default function CollectibleDefaultImage({
   name,
   tokenId,
   handleImageClick,
+  handleImageKeyPress,
 }) {
   const t = useI18nContext();
   return (
     <div
```

Comment:
We should instead make this a `<button>`  so that `onClick` triggers the move.

Reply:
Updated